packages/components/avatar/src/avatar-name.tsx

```diff
@@ -3,10 +3,10 @@ import { useAvatarStyles } from "./avatar-context"
 import { AvatarOptions } from "./avatar-types"
 
 export function initials(name: string) {
-  const [firstName, lastName] = name.split(" ")
```

Comment:
Could change this to:

```ts
const [firstName, ...otherNames] = name.split(" ");
const [lastName] = otherNames.reverse();
```

and keep the rest of the function as-is

Reply:
Thank Luke! Looks even better!